website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 11
+title: Custom Podman machine
+description: Podman Desktop can assist you to install custom Podman machine on Windows and macOS.
+tags: [podman-desktop, podman, installing, windows, macOS]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos, podman machine]
+---
+
+# Creating a Podman machine with Podman Desktop
+
+On macOS and Windows, running the Podman container engine requires running a Linux distribution on a virtual machine.
+
+By default, Podman Desktop is initializing a Podman machine with a standard configuration.
+
+Consider creating a custom Podman machine to:
```

Comment:
❓ 

This section is a bit confusion to the reader. It's self explanatory, maybe a picture would say a thousand words instead?